### ISSUE TITLE
[Feature Request]: Add translation directly within the popup for all available languages 

### ISSUE THREAD

**Issue by beshoo** [laravel-idea/plugin#1030]
### Feature Description

Dear developer.

Thank you for this function 
![image](https://github.com/user-attachments/assets/f7c2bee0-f295-4c74-8912-44838b50ff9d)

The problem is adding the translation only in the eng file
what if I have more than one language, it will be very helpful to add the translation in the popup of the key window, that will save a lot of time.

and if you can add a window to show all translations in one place and we can update then in the same window.

**Comment by beshoo** [comment]
https://github.com/laravel-idea/plugin/issues/855

**Comment by beshoo** [comment]
@adelf any plan to add this ?

**Comment by adelf** [comment]
We are going to release the new plugin, which will cover this need, this or next week. I'll post here about it.

**Comment by beshoo** [comment]
Is it part of laravel idea.? Or a new plug-in ?

Regard

**Comment by adelf** [comment]
The new one. Free with some premium features. The free features will cover this use case.

**Comment by beshoo** [comment]
But will that create conflict with php laravel idea ?

Creating the key...etc

**Comment by adelf** [comment]
No. I hope the "Add key" feature will soon be moved to the new plugin.

**Comment by gorbunov** [comment]
Check new plugin for this and many other improved features for translations support: https://locale-sphere.com
